src/Faithlife.Data/DbConnector.cs

```diff
@@ -132,6 +133,54 @@ public abstract class DbConnector : IDisposable, IAsyncDisposable
 		/// <param name="parameters">The command parameters.</param>
 		public DbConnectorCommand Command(string text, params (string Name, object? Value)[] parameters) => Command(text, DbParameters.Create(parameters));
 
+		/// <summary>
+		/// Creates a new command.
+		/// </summary>
+		/// <param name="sql">The parameterized SQL.</param>
+		public DbConnectorCommand Command(Sql sql)
+		{
+			var (sqlText, sqlParameters) = SqlSyntax.Default.Render(sql);
+			return Command(sqlText, sqlParameters);
+		}
+
+		/// <summary>
+		/// Creates a new command.
+		/// </summary>
+		/// <param name="sql">The parameterized SQL.</param>
+		/// <param name="parameters">Additional command parameters.</param>
+		public DbConnectorCommand Command(Sql sql, DbParameters parameters)
+		{
+			var (text, sqlParameters) = SqlSyntax.Default.Render(sql);
+			return Command(text, sqlParameters.Add(parameters));
+		}
+
+		/// <summary>
+		/// Creates a new command.
+		/// </summary>
+		/// <param name="sql">The parameterized SQL.</param>
+		/// <param name="parameters">Additional command parameters.</param>
+		public DbConnectorCommand Command(Sql sql, params (string Name, object? Value)[] parameters) => Command(sql, DbParameters.Create(parameters));
+
+		/// <summary>
+		/// Creates a new command.
+		/// </summary>
+		/// <param name="sql">The parameterized SQL.</param>
+		public DbConnectorCommand CommandFormat(FormattableString sql) => Command(Sql.Format(sql));
```

Comment:
Do these overloads add enough utility to justify the risks around `FormattableString` (i.e., it downgrades to `string`--with injection--too easily)?

Reply:
Not necessarily. [Comparison here.](https://github.com/Faithlife/FaithlifeData/pull/27/files#diff-961e985c94b8fef1d88f367c09d463d48ec23251a23a4eb5cd72efc78b51e304) I'm willing to drop the `CommandFormat` overloads. If someone really wants them they can always write their own extension methods.

Reply:
Yes, this doesn't seem to add a lot of value to Faithlife.Data itself; I don't see the need to provide two very similar ways to do the same thing.